src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.legal.strings

```diff
@@ -9,10 +9,11 @@
+"DeltaOnboarding_DataProcessing_Info_content" = "You do not need to change anything to find out whether you have had possible exposures involving app users in the participating countries and are therefore at risk of infection yourself. You just need to make sure that the exposure logging feature is still enabled. Exposure logging will automatically warn you about any possible exposure involving another user of the Corona-Warn-App or any other official coronavirus app.\n\nIf you have enabled the “Exposure Notifications” functionality on your iPhone, then your iPhone continuously generates random IDs and sends them via Bluetooth so that they can be received by other smartphones near you. Your iPhone, in turn, receives the random IDs of other smartphones. Your own random IDs and those received from other smartphones are recorded by your iPhone and stored there for 14 days.\n\nFor exposure logging, the app downloads up-to-date lists every day of the random IDs of all users who have shared their test result (more precisely: their own random IDs) via their official coronavirus app in order to warn other users. These lists are then compared with the random IDs of other users which have been recorded by your iPhone.
+The app will inform you if it detects a possible exposure. In this case, the app gains access to the data recorded by your iPhone about the possible exposure (date, duration and Bluetooth signal strength of the contact). The Bluetooth signal strength is used to derive the physical distance to the other user (the stronger the signal, the smaller the distance). The app analyses this information in order to calculate your risk of infection and to give you recommendations for what to do next. This analysis is only performed locally on your iPhone.\n\nApart from you, nobody (not even the RKI or the health authorities of participating countries) will know whether a possible exposure has been detected and what risk of infection has been identified for you.\n\nThe app’s privacy notice (including information about the data processing carried out for the transnational exposure logging feature) can be found in the menu under “App Information” > “Data Privacy”.";
```

Comment:
```suggestion
"DeltaOnboarding_DataProcessing_Info_content" = "You do not need to change anything to find out whether you have had possible exposures involving app users in the participating countries and are therefore at risk of infection yourself. You just need to make sure that the exposure logging feature is still enabled. Exposure logging will automatically warn you about any possible exposure involving another user of the Corona-Warn-App or any other official coronavirus app.\n\nIf you have enabled the “Exposure Notifications” functionality on your iPhone, then your iPhone continuously generates random IDs and sends them via Bluetooth so that they can be received by other smartphones near you. Your iPhone, in turn, receives the random IDs of other smartphones. Your own random IDs and those received from other smartphones are recorded by your iPhone and stored there for 14 days.\n\nFor exposure logging, the app downloads up-to-date lists every day of the random IDs of all users who have shared their test result (more precisely: their own random IDs) via their official coronavirus app in order to warn other users. These lists are then compared with the random IDs of other users which have been recorded by your iPhone. The app will inform you if it detects a possible exposure. In this case, the app gains access to the data recorded by your iPhone about the possible exposure (date, duration and Bluetooth signal strength of the contact). The Bluetooth signal strength is used to derive the physical distance to the other user (the stronger the signal, the smaller the distance). The app analyses this information in order to calculate your risk of infection and to give you recommendations for what to do next. This analysis is only performed locally on your iPhone.\n\nApart from you, nobody (not even the RKI or the health authorities of participating countries) will know whether a possible exposure has been detected and what risk of infection has been identified for you.\n\nThe app’s privacy notice (including information about the data processing carried out for the transnational exposure logging feature) can be found in the menu under “App Information” > “Data Privacy”.";
```